brew_view/static/src/js/run.js

```diff
@@ -33,7 +33,7 @@ export function appRun($rootScope, $state, $stateParams, $cookies, UtilityServic
   });
 
   // Use the SystemService to build the side bar.
-  SystemService.getSystems().then(function(response) {
+  SystemService.getSystems(undefined, 'id,name,version').then(function(response) {
```

Comment:
I don't think we should be setting `includeCommands` to `undefined`. `true` sounds better.

Reply:
I can change this once the backend changes. It's still going to be a little weird then, because it'll basically be `include_commands=True` but commands won't be included in the response.